### PR TITLE
Image changes and menu item details removed 

### PR DESCRIPTION
**Pull request by asha-gadde**

### CODE REVIEW

File: source/Features.rst [hunk 1]
```diff
@@ -171,11 +165,11 @@ There is a contextual menu which is at the right side of the card. Once clicked
 Audit Trail
 ^^^^^^^^^^^
 
-We have brought in an  Audit Trail screen for the Administrator to view security-related audits. Click on the “ ≣ “ option which is available on the left side header. You can see the  following details:
+As an Administrator you can use the **Audit Trail** screen to view security-related audits. Click on the “☰” option which is available on the left side header.
    
 .. image:: images/Audit1.png
 
-Click on the **Audit Trail** menu item to navigate to the Audit Trail page.
+Click on the **Audit Trail** menu item. Through this, you can navigate to the Budget details page.
```
Comment (RLIndia) [inline source/Features.rst:172]: Audit Trail menu item should say it navigates to Audit Trail page, not Budget Details page.

File: source/Features.rst [hunk 2]
```diff
@@ -70,14 +64,14 @@ When Consumed Budget exceeds a threshold (say 80%), the budget management screen
...
-You can  also see researcher-wise budget details which are linked to a particular project and  you can see configured product  details in product-wise budget details page.
+You can  also see researcher-wise budget details which are linked to a particular project and  you can see configured product details in product-wise budget details page.
  
 
 .. image:: images/Image4.png
 
 Users
 ^^^^^
-As an Administrator you can use the "Users" screen to view all users across Research Gateway. Click on the “ ≣ “ option which is available on the left side header. You can see the  following details:
+As an Administrator you can see the "Users" screen to view all users across Research Gateway. Click on the “☰” option which is available on the left side header.
```
Comment (RLIndia) [inline source/Features.rst:74]: replace "you can see" with "you can use"

File: source/Features.rst [hunk 3]
```diff
@@ -105,7 +99,7 @@ You can filter by O.U, Filter by role(Admin/Researcher/Principal Investigator),
 .. image:: images/filter.png
 .. image:: images/sort.png
 
-We have an active filter: enable and disable option. You can toggle between active or all users.
+The user can see an active filter with enable and disable option. You can toggle between active or all users.
```
Comment (RLIndia) [inline source/Features.rst:102]: Use "options" (plural).
Also replace "You can toggle between active or all users." with "You can toggle the view between active or all users."

File: source/Features.rst [hunk 4]
```diff
@@ -338,20 +328,15 @@ The project status changed to “Stopped”. In a Stopped state all underlying r
...
+As a Principal Investigator, you can view the organization-wide budgets from the **Budgets** screen with drill-down to the project, researcher and product level.
 
 **Navigation to Budget screen**
 
-Logged as Principal Account. Click on “☰” Symbol which is available  on left side header. You can see menu like: 
-
-  a. **My projects** : Through this, you can navigate to My Projects page.
-
-  b. **Budget**  : Through this, you can navigate to the Budget Details page. 
+Logged as Principal Account. Click on “☰” Symbol which is available on left side header. Click on the "Budgets" menu item through this, you can navigate to the Budget Details page.
```
Comment (RLIndia) [inline source/Features.rst:339]: Replace "Logged as Principal Account" with "Sign in as the Principal Investigator"

File: source/Features.rst [hunk 5]
```diff
@@ -799,7 +782,7 @@ You can terminate the study through the “Terminate” option.
 
 Public Study
 ------------
-A principal may create a data-store. The “Study” details page will show a tabbed area with the following tabs:
+As a principal Invetigator you can create a data-store. The “Study” details page will show a tabbed area with the following tabs:
```
Comment (RLIndia) [inline source/Features.rst:785]: I thought we did not actually implement "Study creation" screens. ??

Comment (RLIndia) [reply source/Features.rst:785]: Also need to correct spelling of "Investigator"

File: source/GettingStarted.rst [hunk 6]
```diff
@@ -28,7 +28,7 @@ Users with the Administrator role can perform the steps below.
 Users with the Principal Investigator role can perform the steps below.
   
   * `Adding a New Project`_
-  *  Assigning Researchers to projects
+  * `Assigning Researchers to projects`_
```
Comment (RLIndia) [inline source/GettingStarted.rst:31]: Does this hyper-link work? I removed the hyper-link because it was not leading anywhere.

**Review by RLIndia — CHANGES_REQUESTED** [review]

File: source/GettingStarted.rst (hunk 6 above)
Comment (asha-gadde) [reply source/GettingStarted.rst:31]: I added data for that. So I make it a link.

**Review by RLIndia — APPROVED** [review]

**Pull request merged**